cms/static/js/views/pages/group_configurations.js

```diff
@@ -14,18 +19,38 @@ function($, _, gettext, BasePage, GroupConfigurationsListView, ContentGroupListV
                     collection: this.experimentGroupConfigurations
                 });
             }
-            this.contentGroupConfiguration = options.contentGroupConfiguration;
-            this.cohortGroupsListView = new ContentGroupListView({
-                collection: this.contentGroupConfiguration.get('groups')
-            });
+
+            this.allGroupConfigurations = options.allGroupConfigurations || [];
+            this.allGroupViewList = [];
+            for (i = 0; i < this.allGroupConfigurations.length; i++) {
+                restrictEditing = false;
+                if (this.allGroupConfigurations[i].get('scheme') === this.ENROLLMENT_SCHEME) {
+                    restrictEditing = true;
+                }
+                this.allGroupViewList.push(
+                    new ContentGroupListView({
+                        collection: this.allGroupConfigurations[i].get('groups'),
+                        restrictEditing: restrictEditing,
```

Comment:
You can simplify this by ditching the variable restrictEditing all together, and doing this:

````
restrictEditing: this.allGroupConfigurations[i].get('scheme') === this.ENROLLMENT_SCHEME